srcpkgs/toot/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'toot'
+pkgname=toot
+version=0.19.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3-devel python3-setuptools"
```

Comment:
python3-devel shouldn't be in hostmakedepends most of the time.

Reply:
To allow cross compilation, the python-devel package (for python 2.7) must be added to hostmakedepends and makedepends

That is from Manual.md on the github page, which is why I did it; will test without